package.json

```diff
@@ -53,7 +53,8 @@
     "textlint-rule-common-misspellings": "~1.0.1",
     "textlint-rule-no-dead-link": "~4.4.1",
     "textlint-rule-terminology": "~1.1.30",
-    "textlint-rule-write-good": "~1.6.2"
+    "textlint-rule-write-good": "~1.6.2",
+    "snyk": "^1.145.0"
```

Comment:
For stability reasons, we prefer to use ~ instead of ^. Please, use the former ones in the added dependecies.